pyvista/core/pointset.py

```diff
@@ -1510,19 +1509,19 @@ def point_normals(self) -> pyvista.pyvista_ndarray:  # numpydoc ignore=RT01
         --------
         >>> import pyvista as pv
         >>> sphere = pv.Sphere()
-        >>> sphere.point_normals  # doctest:+SKIP
-        pyvista_ndarray([[-2.48721432e-10, -1.08815623e-09, -1.00000000e+00],
-                         [-2.48721432e-10, -1.08815623e-09,  1.00000000e+00],
-                         [-1.18888125e-01,  3.40539310e-03, -9.92901802e-01],
+        >>> sphere.point_normals
+        pyvista_ndarray([[ 0.        ,  0.        ,  1.        ],
+                         [ 0.        ,  0.        , -1.        ],
+                         [ 0.10811902,  0.        ,  0.99413794],
```

Comment:
Note for reviewers: the sign (and accuracy) of the normals printed in the docstrings have changed. This may be partly caused by this PR, since the sphere source [generates its own normals by default](https://vtk.org/doc/nightly/html/classvtkSphereSource.html#a6a12de8cd9281c6791f37ecdaf1ce588), meaning that the default generated normals by vtkSphereSource are printed now instead of being re-computed by `compute_normals`. However, this is also likely caused by earlier bug fixes to `sphere`, e.g. #4674, which would explain the changes in sign as well as changes to the accuracy (e.g. normals are exactly `0` now instead of `1e-10`)

Reply:
There was no merge conflict with #6069 so this should be resolved